tests/unittests/test_request_timeout_param_value.py

```diff
@@ -0,0 +1,214 @@
+import unittest
+from unittest import mock
+from tap_mixpanel.streams import  Revenue
+from tap_mixpanel.client import MixpanelClient
+from tap_mixpanel.__init__ import main
+
+CONFIG = {
+    "api_secret": "dummy_secret",
+    "date_window_size": "30",
+    "attribution_window": "5",
+    "project_timezone": "Europe/Amsterdam",
+    "select_properties_by_default": "true",
+    "start_date": "2020-02-01T00:00:00Z",
+    "user_agent": "tap-mixpanel <api_user_email@your_company.com>",
+    "eu_residency": False,
+    "end_date": "2020-03-02T00:00:00Z"
+}
+
+REQUEST_TIMEOUT = 300
+REQUEST_TIMEOUT_FLOAT = 300.0
+
+class MockCatalog():
```

Comment:
Please add a comment for all mock classes.

Reply:
Added